### PR TITLE
@lennamau: created server, index, knexfile, did migrations and seeds

### PR DESCRIPTION
**Pull request by arunabenji29**

### CODE REVIEW

**Comment by arunabenji29** [comment]
@lennamau : MVP and stretch completed

**Comment by lennamau** [comment]
# Web Testing Challenge
## MVP
- [x]  Post and Get endpoints
- [x]  3 tests per endpoint

## Stretch
- [ ] Validate title is unique -status 405 test
- [x] GET /games/:id endpoint 404 status
- [x] DELETE /games/:id endpoint 404 Status

## What Looks Great
- Awesome job completing MVP and stretch Aruna!
- Clean, uniform code
- Good file structure
- Good job with frequent commits and descriptive messages

## What Could Be Improved
- Comments in code to describe what it is doing

## Rating (1-3)
### 3

File: boards/boards-router.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        // console.log('all games ',board)
+        res.status(200).json({board:board});
+    })
+    .catch(error => {
+        res.status(500).json(error);
+    }
+
+    )
+})
+
```
Comment (lennamau) [inline boards/boards-router.js:18]: ✔

File: boards/boards-router.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            res.status(404).json({message:'Board id not found'});
+        }
+    })
+    .catch(error => {
+        res.status(500).json(error);
+    }
+
+    )
+})
+
```
Comment (lennamau) [inline boards/boards-router.js:36]: ✔

File: boards/boards-router.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        res.status(201).json(ids);
+        }
+    })
+    .catch(error => {
+        res.status(500).json(error);
+    }
+
+    )
+})
+
```
Comment (lennamau) [inline boards/boards-router.js:55]: ✔

File: boards/boards-router.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    })
+    .catch(error => {
+        // console.log('update error ',error)
+        res.status(500).json(error);
+    }
+
+    )
+})
+
+module.exports = router
```
Comment (lennamau) [inline boards/boards-router.js:78]: ✔

File: boards/boards-router.spec.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+    it('responds with 404 OK',async () => {
+    
+        
+        await supertest(server)
+        .delete('/games/1')
+        .expect(404)
+    })
+})
+})
```
Comment (lennamau) [inline boards/boards-router.spec.js:143]: ✔

**Pull request merged**